src/Resources/contao/helper/functions.php

```diff
@@ -15,9 +15,9 @@
  * @param string $strMessage
  * @param string $strLog
  */
-function log_message($strMessage, $strLog='error.log')
```

Comment:
Imho this is wrong. It should be marked `@deprecated` and also use the logger service internally, no?